textile/chat-features.textile

```diff
@@ -518,7 +512,7 @@ h4(#rest-sending-messages-request-v1). Corresponding Realtime Event V1 @(depreca
 h4(#rest-sending-messages-request-v2). Request V2
 
 Below is the full REST payload format for the V2 endpoint. The @metadata@ and @headers@ keys are optional.
-
+Note that @metadata@ is of type map<string, any> or jsonObject.
```

Comment:
We probably don't need to specify `jsonObject` here - `map<string, any>` covers everything?